deploy/podman/pod_ssl.yml

```diff
@@ -0,0 +1,63 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: assisted-installer
+  name: assisted-installer
+spec:
+  containers:
+    - args:
+        - run-postgresql
+      image: quay.io/centos7/postgresql-12-centos7:latest
+      name: db
+      envFrom:
+        - configMapRef:
+            name: config
+      volumeMounts:
+        - mountPath: /var/lib/pgsql
+          name: pg-data
+    - image: docker.io/nmagnezi/assisted-installer-ui:ssl5
```

Comment:
This should probably change, right?

I guess this depends on some other change in the UI?

Reply:
Yes, left it here by mistake. This is the URL for the test image I used with https://github.com/openshift-assisted/assisted-ui/pull/602 before it got merged. I'll change that.

Reply:
Done.